docs/UserGuide.md

```diff
@@ -544,6 +545,12 @@ Listed here are some frequently asked questions. Should you require any other as
 **Q**: Why are there no guests in my GuestBook?<br>
 **A**: If you didn't enter the `clear` command, it is likely that your `guestbook.json` file is **corrupted**, causing GuestBook to reset it to prevent any malicious files from being used.
 
+**Q**: I cannot find a guest staying on a particular date even though it should be within the dates of stay. Why is that so?
+**A**: It is likely that you entered a date within the start and end dates of the guest's stay (e.g, for a guest staying from 13-01-2022 to 15-01-2022, you entered 14-01-2022 to try and find the guest). Currently, this is **not supported** in GuestBook.
```

Comment:
Should we say "Currently, this feature is not supported by GuestBook."?

Reply:
Agreed!